test/core.js

```diff
@@ -670,7 +681,7 @@ describe('snabbdom', function() {
           {remove: function(_, rm) { rm2 = rm; }},
         ]);
         var vnode1 = h('div', [h('a', {hook: {remove: function(_, rm) { rm3 = rm; }}})]);
-	var vnode2 = h('div', []);
+        var vnode2 = h('div', []);
```

Comment:
This was my editor that automatically converted the (previously) tab to spaces. But it's anyway good to avoid mixed tabs and spaces.